app/forms/select_email_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class SelectEmailForm
+  include ActiveModel::Model
+  include ActionView::Helpers::TranslationHelper
+
+  attr_reader :user, :selected_email
+
+  validate :owns_selected_email
+
+  def initialize(user)
+    @user = user
+  end
+
+  def owns_selected_email
+    user_selected_email = EmailAddress.find_with_email(selected_email)
```

Comment:
- I feel like we should be working with IDs, not raw email address strings. This would also limit the amount of work needed to lookup the email, since looking up an email by its string is [not always trivial](https://github.com/18F/identity-idp/blob/66b1b4e6a5ba19ba6c3fecca3358daab72c41445/app/models/email_address.rb#L40-L42).
- We should also validate that the email address is confirmed, per previous review comments.

Reply:
that makes sense. I have refactored to use ID